docs/available-commands.md

```diff
@@ -81,6 +81,10 @@ Sends text to a named terminal, which is then executed in the terminal. If the n
 
 Note: In order to send multiple commands spread out across multiple links in the same terminal window, ensure that the `name` passed is consistent.
 
+## vscode.didact.sendNamedTerminalAStringNoLF
```

Comment:
the name sounds very technical. What means "LF"?

Reply:
NoLF = No Linefeed, which corresponds to the vscode terminal's "sendText" and its "addNewLine" argument

See https://code.visualstudio.com/api/references/vscode-api#Terminal